### ISSUE TITLE
Update the README

### ISSUE THREAD

**Issue by p2r3** [p2r3/epochtal#23]
The contents of the README.md document are currently Bun defaults, plus a leftover to-do list from development. This should instead briefly describe the project and document how to fully set up the system with either empty placeholder boards or an example from the weekly tournament archives.

**Comment by p2r3** [comment]
Partially resolved in d887d41. Marking as closed, as committing placeholder boards to the repository should be a separate issue after #29 is resolved.
